Extensions/one_click_postage.css

```diff
@@ -11,6 +11,19 @@
     line-height: 10px;
 }
 
+.post.themed_posts_experiment .post_wrapper,
+.post.themed_posts_experiment.post_full .post_footer,
+.post.themed_posts_experiment .post_controls,
+.post.themed_posts_experiment .post_controls_inner,
+.post.themed_posts_experiment .reblogged {
+	background-color: inherit;
```

Comment:
Probably should just move these to the original selectors, rather then scoping them to `themed_posts_experiment` (i.e. so we accommodate custom themes as well)